### PR TITLE
Byebye torch 1.10

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Byebye torch 1.10 ☃️

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
Nice 🧹

File: tests/models/unispeech_sat/test_modeling_unispeech_sat.py
[the remark this answers is not in the document]
Comment (ydshieh) [reply tests/models/unispeech_sat/test_modeling_unispeech_sat.py:1]: Sorry, which tolerances you mean? I am not able to see the line

**Review by LysandreJik — APPROVED** [review]

**Comment by ydshieh** [comment]
OK, going to merge. Feel free to drop a more detailed comment on tolerances  if it's something needs to be fixed.

File: tests/models/unispeech_sat/test_modeling_unispeech_sat.py
[the remark this answers is not in the document]
Comment (ArthurZucker) [reply tests/models/unispeech_sat/test_modeling_unispeech_sat.py:1]: Just the TODO said `update the tolerance after the CI moves to torch 1.10`

[the remark this answers is not in the document]
Comment (ydshieh) [reply tests/models/unispeech_sat/test_modeling_unispeech_sat.py:1]: oh yes. Sorry. However I don't have the full context about this ... and we need @anton-l for the change if necessary

**Pull request merged**